go.mod

```diff
@@ -35,7 +35,7 @@ require (
 	go.mongodb.org/atlas-sdk/v20240530005 v20240530005.0.0
 	go.mongodb.org/atlas-sdk/v20240805005 v20240805005.0.0
 	go.mongodb.org/atlas-sdk/v20241113005 v20241113005.0.0
-	go.mongodb.org/atlas-sdk/v20250312001 v20250312001.0.0
```

Comment:
small downside of this is we will have failing snyk PR checks in the PoC dev branch due to `Failed to process go.mod`